documentation/book/con-upgrade-intro.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
```

Comment:
it's just a comment but could be miss leading, it should be `assembly-upgrade.adoc`

Reply:
Thanks Paolo. Updated.